src/popup/router/components/FormTextarea.vue

```diff
@@ -40,12 +41,20 @@ export default {
     value: { type: String, default: '' },
     error: Boolean,
     placeholder: { type: String, default: '' },
+    enterSubmit: Boolean,
     size: {
       type: String,
       default: null,
       validator: (val) => SIZES.includes(val),
     },
   },
+  methods: {
+    handleEnterClick(event) {
+      if (!this.enterSubmit) return;
+      this.$emit('submit');
+      event.preventDefault();
```

Comment:
I think it's better to:
```
if (this.enterSubmit) {
    this.$emit('submit');
    event.preventDefault();
}
```

Reply:
I think otherwise, you end the function execution when it's not matching the conditions